image_analysis/service.py

```
@@ -0,0 +1,158 @@
+            "created_at": datetime.now(timezone.utc),
+        }
+
+        try:
+            await asyncio.wait_for(_job_queue.put((job_id, user_id, img, token, appcheck_token)), timeout=10.0)
+        except asyncio.TimeoutError:
+            _job_store.pop(job_id, None)
+            _active_users.discard(user_id)
+            return JSONResponse(status_code=504, content={"error": "Server is busy"})
+
+        return JSONResponse(status_code=202, content={"job_id": job_id})
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)